docs/topics/logging.rst

```diff
@@ -242,6 +242,23 @@ e.g. in the spider's ``__init__`` method::
 If you run this spider again then INFO messages from
 ``scrapy.spidermiddlewares.httperror`` logger will be gone.
 
+You could also filter logs based on a substring. This can be achieved using `Filter` class. For example, if you want to filter log message that contains a string 'filter', you can do it like this::
+
+    import logging
+
+    class ContentFilter(logging.Filter):
+        def filter(self, record):
+            return 'filter' in record.msg
+
+    logger = logging.getLogger('mycustomlogger')
```

Comment:
I think we should make it clear how to grab the root logger that Scrapy installs.

Reply:
@Gallaecio, thank you for the feedback, I will look into this next week!

Thank you

Reply:
@Gallaecio, sorry for this late reply, i had exams at the moment. so by root logger do you mean calling `logging.getLogger()` instead of `logging.getLogger('mycustomlogger')` as in the above example? could you please elaborate on this?

Reply:
I’m not sure if `logging.getLogger()` would work, or if you would need to access something else that is not a logger. The point is that we should document an approach that can be used for:
- The whole project. If a matching message is logger by a middleware, a spider, or something else, it should be filtered out. I’m not sure if the same steps using `getLogger()` would work, or something else is needed. Try.
- A specific logger (i.e. from a specific middleware or from a specific spider). If this is as simple as using `getLogger` with the right parameter, I think the documentation just needs to show an example of logging and indicate what is the logger name there (for Scrapy users not familiar yet with the basics of Python logging).

Reply:
@Gallaecio , i now get your point and it is valid. am not sure how either to get the root logger for the whole project. trying to look into it.